File: problemas/iniciante/1006/1006.r
input <- file('stdin', 'r')

a <- as.double(readLines(input, n=1))
b <- as.double(readLines(input, n=1))
c <- as.double(readLines(input, n=1))

write(paste("MEDIA =", format(round((2 * a + 3 * b + 5 * c)/10, 1), nsmall=1)), '')

File: problemas/iniciante/1006/1006.dart
import 'dart:io';

void main()
{
    double a = double.parse(stdin.readLineSync()!);
    double b = double.parse(stdin.readLineSync()!);
    double c = double.parse(stdin.readLineSync()!);

    print("MEDIA = ${((2 * a + 3 * b + 5 * c)/10).toStringAsFixed(1)}");
}